packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -552,7 +552,7 @@ class DeepLinksController extends DisposableController
           false);
 
       if (linkdata.os.contains(PlatformOS.ios)) {
-        final List<String> iosPaths = iosDomainPaths[linkdata.domain] ?? [];
+        final iosPaths = iosDomainPaths[linkdata.domain] ?? [];
```

Comment:
add the type to the empty list or else this would be dynamically typed when `iosDomainPaths[linkdata.domain]` is null